### PR TITLE
suggest images

### LINKED ISSUE #81: Suggest existing images in image selection

### PR DESCRIPTION
**Pull request by elKei24**
Closes #81

### CODE REVIEW

File: scripts/realm/getAvailableImages.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+exports = async function() {
+    if (!context.functions.execute("checkIsFrontendAdmin")) {
+        return "Unauthorized";
```
Comment (cstanger) [inline scripts/realm/getAvailableImages.js:3]: is is also possible to send a unauthorized status code as response ? I didn't find anything - but as this is only for the admin view - I would stop putting effort in here

File: scripts/realm/getAvailableImages.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    }
+    const s3 = context.services.get('AWS').s3("eu-central-1");
+    return await s3.ListObjectsV2({
```
Comment (cstanger) [inline scripts/realm/getAvailableImages.js:6]: nice!

**Comment by cstanger** [comment]
Nice Work @elKei24 
I would not have implemented the add service action - and instead just re-calling the Realm function for getting all image paths  again after uploading an image - not most efficient way but less effort! But thank you for your extra mile!!! GREAT

File: scripts/realm/getAvailableImages.js (hunk 1 above)
Comment (elKei24) [reply scripts/realm/getAvailableImages.js:3]: Could not find anything neither, but would definitively be useful 👍

**Comment by elKei24** [comment]
> Nice Work @elKei24
> I would not have implemented the add service action - and instead just re-calling the Realm function for getting all image paths again after uploading an image - not most efficient way but less effort! But thank you for your extra mile!!! GREAT

Well, isn't that what I did? 😄

**Pull request merged**